src/common/config_opts.h

```diff
@@ -1410,6 +1410,8 @@ OPTION(rgw_period_push_interval_max, OPT_DOUBLE, 30) // maximum interval after e
 
 OPTION(rgw_swift_versioning_enabled, OPT_BOOL, false) // whether swift object versioning feature is enabled
 
+OPTION(rgw_min_read_ahead, OPT_INT, 1000) // minimum number of entries to read from rados
```

Comment:
call this rgw_min_keys_read ? the current name sounds like you're doing read-ahead of data.